titus-api/src/main/java/com/netflix/titus/api/jobmanager/model/job/disruptionbudget/RatePerIntervalDisruptionBudgetRate.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.titus.api.jobmanager.model.job.disruptionbudget;
+
+import java.time.Duration;
+import java.util.Objects;
+import javax.validation.constraints.Min;
+
+public class RatePerIntervalDisruptionBudgetRate extends DisruptionBudgetRate {
+
+    @Min(1)
```

Comment:
I think there should be maxes on this.

Reply:
Max makes sense for interval. For `limitPerInterval` it is ok to have it as high as user wants it.
I will set `intervalMs` max to 1 hour.